zebra-rpc/src/methods/get_block_template_rpcs/types/validate_address.rs

```diff
@@ -0,0 +1,39 @@
+//! Response type for the `validateaddress` RPC.
+
+use crate::methods::get_block_template_rpcs::types::hex_data::HexData;
+
+/// `validateaddress` response
+#[derive(Clone, Default, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+pub struct Response {
+    /// Whether the address is valid.
+    ///
+    /// If not, this is the only property returned.
+    #[serde(rename = "isvalid")]
+    pub is_valid: bool,
+
+    /// The zcash address that has been validated.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub address: Option<String>,
+
+    /// The hex encoded scriptPubKey generated by the address.
+    #[serde(rename = "scriptPubKey")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub script_pub_key: Option<HexData>,
+
+    /// The hex value of the raw public key.
+    #[serde(rename = "pubkey")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub pub_key: Option<HexData>,
```

Comment:
Nitpick: If miners don't need these fields, can we leave them out of the code for now?

Reply:
Okay. Removed in [`f452c89`](https://github.com/ZcashFoundation/zebra/pull/6086/commits/f452c89d645f4b3edfda0478c9460bddb2e81f38).